mode/src/processing/mode/android/AndroidSDK.java

```diff
@@ -501,15 +502,26 @@ static public AndroidSDK download(final Frame editor, final AndroidMode androidM
     
     final int result = showSDKLicenseDialog(editor);
     if (result == JOptionPane.YES_OPTION) {
-      sdk.acceptLicenses();
-      String msg = AndroidMode.getTextString("android_sdk.dialog.sdk_installed_body", PROCESSING_FOR_ANDROID_URL, WHATS_NEW_URL);
-      File driver = AndroidSDK.getGoogleDriverFolder();
-      if (Platform.isWindows() && driver.exists()) {
-        msg += AndroidMode.getTextString("android_sdk.dialog.install_usb_driver", DRIVER_INSTALL_URL, driver.getAbsolutePath()); 
-      }
-      AndroidUtil.showMessage(AndroidMode.getTextString("android_sdk.dialog.sdk_installed_title"), msg);      
+     sdk.acceptLicenses();
+     File sdkManager = findCliTool(new File(sdk.getCommandLineToolsFolder(), "bin"), "sdkmanager");
+     try {
+    	 String[] cmdToDownloadEmu = {sdkManager.getCanonicalPath(),"emulator"};
+      Process process = Runtime.getRuntime().exec(cmdToDownloadEmu, null, sdk.getFolder());
+      new RedirectStreamHandler(new PrintWriter(System.out, true), process.getInputStream());
+      new RedirectStreamHandler(new PrintWriter(System.out, true), process.getErrorStream());
+      int emulatorDownloadResultCode = process.waitFor();
+      System.out.println("*-*-*-*itsmylog:EmulatorResultCode:"+emulatorDownloadResultCode);
```

Comment:
please change the log messages and dont use special signs in logs !

Reply:
Sure, I will remove these logs for testing in the next commits, Thanks for pointing Aditya